modules/codegen/src/test/scala/support/SwaggerSpecRunner.scala

```diff
@@ -31,12 +32,13 @@ trait SwaggerSpecRunner extends EitherValues {
   ): (Context, FunctionK[CodegenApplication[L, ?], Target]) => (ProtocolDefinitions[L], Clients[L], Servers[L]) = {
     val parseOpts = new ParseOptions
     parseOpts.setResolve(true)
+    implicitly[ScalaTerms[L, Free[CodegenApplication[L, ?], ?]]]
     runSwagger(new OpenAPIParser().readContents(spec, new java.util.LinkedList(), parseOpts).getOpenAPI, dtoPackage) _
   }
 
   def runSwagger[L <: LA](swagger: OpenAPI, dtoPackage: List[String] = List.empty)(context: Context, framework: FunctionK[CodegenApplication[L, ?], Target])(
       implicit Fw: FrameworkTerms[L, CodegenApplication[L, ?]],
-      Sc: ScalaTerms[L, CodegenApplication[L, ?]],
+      Sc: ScalaTerms[L, Free[CodegenApplication[L, ?], ?]],
```

Comment:
`Free` is being _added_ here?

Reply:
so, this was the really exciting part of this PR. Take a look at the diff of the type of `ScalaTerms`:
```diff
-class ScalaTerms[L <: LA, F[_]](implicit I: InjectK[ScalaTerm[L, ?], F]) {
+abstract class ScalaTerms[L <: LA, G[_]] {
  // Changed F[_] to G[_] to disambiguate this example
```
... as well as the diff of the definition of one of the members...
```diff
-  def litString(value: String): Free[F, L#Term]                                 = Free.inject[ScalaTerm[L, ?], F](LitString(value))
+  def litString(value: String): G[L#Term]
```
... as well as the syntax injector, relying on `InjectK` to lift the terms of this interface into the target program:
```diff
-  implicit def scalaTerm[L <: LA, F[_]](implicit I: InjectK[ScalaTerm[L, ?], F]): ScalaTerms[L, F] = new ScalaTerms[L, F]
 // No bespoke implementation of functions needed, since ScalaTerms wasn't abstract before
+  implicit def scalaTerm[L <: LA, F[_]](implicit I: InjectK[ScalaTerm[L, ?], F]): ScalaTerms[L, Free[F, ?]] = new ScalaTerms[L, Free[F, ?]] {
+    def litString(value: String): Free[F, L#Term] = Free.inject[ScalaTerm[L, ?], F](LitString(value))
```

From the consumer's perspective, `implicit def scalaTerm` actually has the same structure, and can be used in the same way, because if we pass `Free[F, ?]` into `ScalaTerm`'s constructor in place of `G[_]`, it turns out all the types just line right back up again.

```scala
def litString(value: String): G[L#Term] // ... becomes...
def litString(value: String): Free[F, L#Term] // which was the exact same type as the first thing.
```

At the _call sites,_ however, anywhere we're still depending on `Free` to do our job still actually need to be explicit that they're using the `ScalaTerms[L, Free[CodegenApplication[L, ?], ?]]` implementation during this transitional period, only until there are implementations of all depended interfaces for the desired `F[_]` (`Target[_]` in this case) for any given function.

You can see that here: https://github.com/twilio/guardrail/blob/fc5e6d921b6a6ad3152a6a4821a9e0d1a8c69bce/modules/codegen/src/main/scala/com/twilio/guardrail/ProtocolGenerator.scala#L676-L678

Even though it doesn't explicitly mention whether it uses `Free[F, ?]` or `Target[_]` or whatever, all the implicit terms match (since there's only one), so _whatever_ `F[_]` gets subbed in will match, since we started providing both definitions.

In this way, the whole codebase will get converted over to this syntax without changing anything, really. The `Free` overrides we're doing on our internal guardrail extensions will continue to work until the very end, when only `Free` is deleted, but no other changes are made, since all the syntax changes have already been made gradually through these PRs. (That's another reason for prioritizing the scalafix rules to automate these changes)